CHANGELOG.md

```diff
@@ -2,21 +2,35 @@
 
 #### Upcoming Changes
 
+<<<<<<< HEAD
```

Comment:
jejeje i think you forgot the conflicts in the Changelog

Reply:
Oops

Reply:
Fixed